### ISSUE TITLE
`stepmod-extract-terms` command generating non-unique bibliographic references

### ISSUE THREAD

**Issue by ronaldtse** [metanorma/stepmod-utils#72]
In 991-generated-bibliography.adoc, `ISO-TS_10303-1015_2004` is duplicated (and many others).

I had to fix it like this in the iso-10303-2 document:
https://github.com/metanorma/iso-10303-2/commit/91b152a08cab0ac0851847d2365f23e16035e539

```
* [[[ISO-TS_10303-1260_2011,ISO/TS 10303-1260:2011]]], _Industrial automation systems and integration -- Product data representation and exchange -- Part 1260: Scheme_
* [[[ISO-TS_10303-1015_2004,ISO/TS 10303-1015:2004]]], _Industrial automation systems and integration -- Product data representation and exchange -- Part 1015: Security classification_
* [[[ISO-TS_10303-1357_2004,ISO/TS 10303-1357:2004]]], _Industrial automation systems and integration -- Product data representation and exchange -- Part 1357: Selected item_
```

```
* [[[ISO-TS_10303-1260_2011,ISO/TS 10303-1260:2011]]], _Industrial automation systems and integration -- Product data representation and exchange -- Part 1260: Scheme_
* [[[ISO-TS_10303-1015_2004,ISO/TS 10303-1015:2004]]], _Industrial automation systems and integration -- Product data representation and exchange -- Part 1015: Security classification_
* [[[ISO-TS_10303-1357_2021,ISO/TS 10303-1357:2021]]], _Industrial automation systems and integration -- Product data representation and exchange -- Part 1357: Selected item_
```

**Comment by ronaldtse** [comment]
How I run this:

1. Go to https://github.com/metanorma/iso-10303-stepmod
2. Run `stepmod-extract-terms .`

**Comment by ronaldtse** [comment]
In this command, the terms also need to be sorted alphabetically.

Probably can be done together with https://github.com/metanorma/stepmod-utils/issues/14

**Comment by ronaldtse** [comment]
This is probably fixed in 08f9d0ee0354c3a17605e683c5ebf36eabb4e115, I can't reproduce it anymore. Thanks @w00lf !
